webapp/package.json

```diff
@@ -13,19 +13,19 @@
   },
   "private": true,
   "dependencies": {
-    "@angular/animations": "^7.2.15",
+    "@angular/animations": "^8.2.10",
     "@angular/cdk": "^7.3.7",
-    "@angular/common": "^7.2.15",
-    "@angular/compiler": "^7.2.15",
-    "@angular/core": "^7.2.15",
+    "@angular/common": "^8.2.10",
+    "@angular/compiler": "^8.2.10",
+    "@angular/core": "^8.2.10",
     "@angular/flex-layout": "^7.0.0-beta.24",
-    "@angular/forms": "^7.2.15",
+    "@angular/forms": "^8.2.10",
     "@angular/material": "^7.3.7",
-    "@angular/platform-browser": "^7.2.15",
-    "@angular/platform-browser-dynamic": "^7.2.15",
-    "@angular/router": "^7.2.15",
+    "@angular/platform-browser": "^8.2.10",
+    "@angular/platform-browser-dynamic": "^8.2.10",
+    "@angular/router": "^8.2.10",
     "@c4dt/cothority": "^3.1.23",
-    "@c4dt/dynacred": "dev",
+    "@c4dt/dynacred": "0.1.1-p20191010083636.0",
```

Comment:
You might want to keep using the "dev" tag (yes, it gets replaced at each update, thanks npm)

Reply:
Right, it's probably the "~automatic Angular upgrade procedure" that changed it...

Reply:
This being said, shouldn't we hardcode the version and upgrade it manually, in order to have reproducible builds?